integration/examples/bazel/WORKSPACE

```diff
@@ -2,20 +2,21 @@ workspace(name = "skaffold")
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
+
 http_archive(
     name = "io_bazel_rules_docker",
-    strip_prefix = "rules_docker-0.7.0",
-    urls = ["https://github.com/bazelbuild/rules_docker/archive/v0.7.0.tar.gz"],
-    sha256 = "aed1c249d4ec8f703edddf35cbe9dfaca0b5f5ea6e4cd9e83e99f3b0d1136c3d",
+    strip_prefix = "rules_docker-0.8.0",
+    urls = ["https://github.com/bazelbuild/rules_docker/archive/v0.8.0.tar.gz"],
+    sha256 = "5647f4f7a6b7c247e788675963e2e03a6e7156e1",
```

Comment:
The sha256 seems wrong.

`            ERROR: An error occurred during the fetch of repository 'io_bazel_rules_docker':
               com.google.devtools.build.lib.syntax.EvalException: Definition of repository io_bazel_rules_docker: Syntactically invalid SHA256 checksum: '5647f4f7a6b7c247e788675963e2e03a6e7156e1' at /go/src/github.com/GoogleContainerTools/skaffold/integration/examples/bazel/WORKSPACE:6:1`

It should be `3556d4972571f288f8c43378295d84ed64fef5b1a875211ee1046f9f6b4258fa`:  https://github.com/bazelbuild/rules_docker#setup

Reply:
Thanks! updated it!